### PR TITLE
Sync track features

### PR DESCRIPTION
**Pull request by danlipsa**

### CODE REVIEW

**Comment by danlipsa** [comment]
@mleotta Please review

File: arrows/mvg/applets/track_features.cxx
Comment (mleotta) [inline arrows/mvg/applets/track_features.cxx:427]: I had forgotten that previously the `camera_frames` was just for restarting previous results that already had cameras.  More commonly, it was for cases where metadata was used to create initial guess for the cameras.  That is, this helps align which frames we track to which frames have metadata.

For now I suggested with initialize `camera_frames` with the frames of `md_map` if metadata is being used or empty if not.  That's not exactly what the GUI does.  The GUI converts metadata to cameras first.  It's possible that some frames with metadata will not have sufficient metadata to make a camera, so you will get fewer camera frames.  However, in most cases, `camera_frames` will be the same as the frames on which we have metadata.

**Review by mleotta — CHANGES_REQUESTED** [review]
This looks like a good step in the right direction, but I have some further suggestions.

**Comment by danlipsa** [comment]
@mleotta Please review. This the MR that addresses your comments and also uses only the selected_frames to trach features. It gets different tracks.txt than the gui though - maybe because of the cameras.

**Review by mleotta — APPROVED** [review]
Looks good to me, but we will have to resolved conflicts with #1101 before both can merge.

There are additional possibly improvements, like loading default config from an installed file and additional command line options to set input and output files, but these can come in a new branch.

**Comment by mleotta** [comment]
Jenkins test this please

**Comment by danlipsa** [comment]
Jenkins test this please

**Comment by danlipsa** [comment]
@mleotta I fixed the conflicts. Is this ready to merge?

**Comment by mleotta** [comment]
Jenkins test this please

**Comment by mleotta** [comment]
@danlipsa I will merge when all the CI tests come back clean.

**Comment by mleotta** [comment]
I'm not sure why travis CI is failing but it seems unrelated to this branch.  I will merge.

**Pull request merged**
